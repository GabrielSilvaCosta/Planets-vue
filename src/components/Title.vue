<template>
    <div class="title">
      <h1>{{ headline }}</h1>
    </div>
  </template>
  
  <script>
  export default {
    name: "Title",
    props: ["headline"],
  };
  </script>
  
  <style scoped>

    .title {
      display: flex;
      justify-content: center;
      align-items: center;
      height: 100px;
      background-color: #333;
      color: #fff;
      font-size: 24px;
      font-weight: bold;
      text-align: center;
      text-transform: uppercase;
      letter-spacing: 4px;
      margin-bottom: 20px;
    }

  </style>
  