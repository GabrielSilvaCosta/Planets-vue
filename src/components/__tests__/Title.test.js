import { mount } from "@vue/test-utils";
import { describe, expect, it } from "vitest";
import Title from "../Title.vue";

describe("Title", () => {
    it("should render correctly", () => {
        const wrapper = mount(Title, {
            props: {
                title: "Teste",
            },
        });
        expect(wrapper.html()).toMatchSnapshot();
    });
})