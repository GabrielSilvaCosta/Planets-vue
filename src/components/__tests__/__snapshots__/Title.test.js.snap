// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Title > should render correctly 1`] = `
"<div data-v-6db1a9b7="" class="title" title="Teste">
  <h1 data-v-6db1a9b7=""></h1>
</div>"
`;
